features/step_definitions/auction_create_steps.rb

```diff
@@ -88,10 +88,14 @@
   @auction.skills << skills
 end
 
-Given(/^there is a budget approved auction$/) do
+Given(/^there is a budget approved auction with bids$/) do
   @auction = FactoryGirl.create(:auction, :c2_approved, :with_bids)
 end
 
+Given(/^there is a budget approved auction$/) do
```

Comment:
yeah but now that we have a similar step "with bids" perhaps renaming this would be a good idea?